src/conditionals/third-party/wpml-wpseo-conditional.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Yoast\WP\SEO\Conditionals\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Conditional;
+
+/**
+ * Conditional that is met when the Yoast SEO Multilingual plugin,
+ * a glue plugin developed by and for WPML, is active.
+ */
+class WPML_WPSEO_Conditional implements Conditional {
+
+	const PATH_TO_WPML_WPSEO_PLUGIN_FILE = 'wp-seo-multilingual/plugin.php';
```

Comment:
Please mark this as `@internal` - See https://github.com/Yoast/developer-docs/blob/a3e2c2336e7de88edde51c206b1453c756ab2ac0/docs/development/standards/constants-usage.md

Because we don't want to everybody use it everywhere.